Games/types/Mafia/roles/cards/KillMonsterVisitors.js

```diff
@@ -0,0 +1,64 @@
+const Card = require("../../Card");
+const { PRIORITY_KILL_MONSTER_VISITORS_ENQUEUE, PRIORITY_KILL_MONSTER_VISITORS } = require("../../const/Priority");
+
+module.exports = class KillMonsterVisitors extends Card {
```

Comment:
this card should really be kill vampire visitors instead of monster visitors

Reply:
okay sure, then keep monsters as the title.
The game still breaks with vampires.